xep-0403.xml

```diff
@@ -207,64 +254,143 @@ A user MAY share presence information with the channel, for one or more online c
       </p>
 
 
 
-    </section3>
+    </section2>
 
-    <section3 topic="User Leaving a Channel" anchor="usecase-presence-leave">
+    <section2 topic="User Leaving a Channel" anchor="usecase-presence-leave">
       <p>
         The primary actions for a user leaving a channel are specified in &xep0369;.    This section sets out additional actions for handling presence.    When a user leaves the channel, all entries for the user's clients MUST be removed from the participants node.   The MIX channel MUST distribute unavailable presence notifications for each client removed to all subscribers of the participants node.
       </p>
 
       <example caption="Channel Distributes Notification when User Leaves Channel">
-        <![CDATA[<presence from='12345#coven@mix.shakespeare.example/UUID-a1j/7533'
+<![CDATA[<presence from='12345#coven@mix.shakespeare.example/UUID-a1j/7533'
           to='hecate@shakespeare.example'
           id='77E07BB0-55CF-4BD4-890E-3F7C0E686BBD'
-          type='unavailable'/>]]></example>
+          type='unavailable'>
+      <mix xmlns='urn:xmpp:presence:0'>
+        <jid>hecate@shakespeare.example/UUID-x4r/2491</jid>
+        <nick>thirdwitch</jid>
+      </mix>
+</presence>]]></example>
 
+    </section2>
 
-    </section3>
 
+<section2 topic="Relaying IQ Stanzas" anchor="usecase-iq-relay">
+  <p>
+    MIX channels MAY relay IQ stanzas between participants.   This is often useful to obtain client information where a direct request to the client would be blocked.   When a client sends an IQ stanza through a MIX channel, it will set the 'from' to its own JID and set the 'to' to the encoded JID of the recipient.    The MIX channel will modify the JIDs in the outgoing message, so that the 'to' is the full JID of the recipient and the 'from' is the encoded JID of the sender.   This is illustrated in the vCard section below.
+  </p>
+</section2>
+  
+  
+  
+  <section2 topic="Requesting a vCard through a Channel" anchor="usecase-vcard">
+    <p>A client  MAY request the vCard of a channel participant through a MIX channel, for example to get an avatar.  The MIX channel MAY pass this request on or MAY block it.  vCard requests MAY use &xep0054; (vcard-temp) or &xep0292; (vCard4 over XMPP).  The MIX channel does not process the vCard requests, but simply relays them on to real bare JID of the target.  A MIX service MAY choose to relay one or both protocols.   Where a MIX service relays  one or both of these protocols, each protocol relayed MUST be advertised as a feature of the MIX service.   In the following example, using vcard-temp, the requesting client sends a message to the  encoded JID of the channel participant for which the vCard is desired.</p>
```

Comment:
This seems out of place. The stuff about advertising vcard support seems wrong (you advertise vcards if you support vcards), and it's just a case of forwarding iqs, so this seems to be saying more than is needed. There's also more MAYs in here than there should be, I think. I don't think this is optional-to-implement.